src/garage/envs/task_id_wrapper.py

```diff
@@ -0,0 +1,54 @@
+"""Wrapper for Metaworld environment to provide properly-named tasks."""
+
+import gym
+
+
+class TaskIdWrapper(gym.Wrapper):
+    """Wrapper for Metaworld environment to provide properly-named tasks."""
+
+    @property
+    def _hidden_env(self):
+        """gym.Env: The underlying Metaworld environment."""
+        env = self.env
+        while hasattr(env, 'env'):
+            env = env.env
+        return env
+
+    @property
+    def task_names(self):
+        """list[str]: List of all available names."""
+        # pylint: disable=protected-access
+        return self._hidden_env._task_names
+
+    def step(self, action):
+        """gym.Env step for the active task env.
+
+        Args:
+            action (np.ndarray): Action performed by the agent in the
+                environment.
+
+        Returns:
+            tuple:
```

Comment:
returning a tuple is equivalent to multiple return values. please use that style instead (it looks just like the Args section)